### PR TITLE
Release 1.4.0 Release Candidate

### PR DESCRIPTION
**Pull request by markmandel**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,90 @@
...
+
+- Update documentation \(examples and website\) to use Kubernetes 1.14 [\#1364](https://github.com/googleforgames/agones/pull/1364) ([roberthbailey](https://github.com/roberthbailey))
+- Update terraform configs to use Kubernetes 1.14 [\#1342](https://github.com/googleforgames/agones/pull/1342) ([roberthbailey](https://github.com/roberthbailey))
+- Update developer tooling to Kubernetes 1.14 [\#1341](https://github.com/googleforgames/agones/pull/1341) ([roberthbailey](https://github.com/roberthbailey))
+- Change the GameServerAllocationPolicy version to stable. [\#1332](https://github.com/googleforgames/agones/pull/1332) ([pooneh-m](https://github.com/pooneh-m))
+- Changing the allocator API to gRPC [\#1314](https://github.com/googleforgames/agones/pull/1314) ([pooneh-m](https://github.com/pooneh-m))
+
+**Implemented enhancements:**
+
+- Change GameServerAllocationPolicy CRD version to stable [\#1290](https://github.com/googleforgames/agones/issues/1290)
```
Comment (pooneh-m) [inline CHANGELOG.md]: This perhaps should not be here? it is mentioned in the Breaking change section.

File: docs/governance/templates/release.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ Images available with this release:
 - [gcr.io/agones-images/udp-server:{example-version}](https://gcr.io/agones-images/udp-server:{example-version})
 - [gcr.io/agones-images/tcp-server:{example-version}](https://gcr.io/agones-images/tcp-server:{example-version})
 - [gcr.io/agones-images/xonotic-example:{example-version}](https://gcr.io/agones-images/xonotic-example:{example-version})
+- [gcr.io/agones-images/supertuxkart-example:{example-version}](https://gcr.io/agones-images/supertuxkart-example:{example-version})
```
Comment (aLekSer) [inline docs/governance/templates/release.md]: There is an option to add `gcr.io/agones-images/crd-client:0.1`

**Comment by markmandel** [comment]
> This perhaps should not be here? it is mentioned in the Breaking change section.

Added the `breaking` tag to the issue, so it'll reorganise itself in the changelog when we regen it for stable :+1:

File: CHANGELOG.md (hunk 1 above)
Comment (markmandel) [reply CHANGELOG.md]: This perhaps should not be here? it is mentioned in the Breaking change section. :+1:

Comment (markmandel) [reply CHANGELOG.md]: I added the `breaking` tag to the issue, so it will move in the Changelog when we regen for the stable release. :+1:

**Pull request merged**